web-client/src/components/code-higlighter.tsx

```diff
@@ -0,0 +1,49 @@
+import { createResource } from "solid-js";
+import {
+  createHighlighter,
+  type HighlighterLang,
+  getHighlightedCode,
+} from "~/lib/code-highlight";
+import sanitizeHtml from "sanitize-html";
```

Comment:
Not completely related to the move PR but a question for @CrabNejonas :

Any reason why `sanitize-html` was chosen over `dompurify`?

The later one is more performant, popular and allows more html tags to be used safely.

Reply:
Not really, just one that I had used before, `dompurify` might be a better choice though